docs/sql/statements/explain.rst

```diff
@@ -19,6 +19,7 @@ Synopsis
 ::
 
     EXPLAIN [ ANALYZE ] statement
+    EXPLAIN [ (ANALYZE | COSTS) (TRUE | FALSE) ] statement
```

Comment:
Couldn't we allow to list several options? `EXPLAIN (ANALYZE true, COSTS true)`, similar to how PostgreSQL does it? 
https://www.postgresql.org/docs/current/sql-explain.html

Reply:
`ANALYZE` returns json while `COSTS` returns the normal logical plan tree including costs, so this does not compose together.

Reply:
What's preventing us from allowing to combine `ANALYZE` and `COSTS` the JSON format itself doesn't prohibit it?

Reply:
So you prefer allowing multiple options simultaneously and throwing a runtime exception when the combination is not allowed over the current approach ? @mfussenegger

Reply:
The two Synopses would be:

```
EXPLAIN [ (ANALYZE | COSTS) (TRUE | FALSE) ] statement
```
vs:
```
EXPLAIN [ ( option [, ...] ) ] statement

where option is:

    ANALYZE [ boolean ]
    COSTS [ boolean ]
```
Note, that all variations of the first version can also be expressed with the second version. So extending later is no problem.